packages/documentation/src/pages/getting-started/internal-tools.mdx

```diff
@@ -1,153 +1,134 @@
 
+```bash
+ls ~/.ssh
+```
+*The list of files should include `id_rsa_vagov` and `id_rsa_vagov.pub`. Only share keys with `.pub` (public) extension; Private keys should never leave your computer.*
 
-#### Configure the SOCKS proxy - for external developers
+## Configure the SOCKS proxy
+Below are configuration instructions for internal and external developers.
 
-1. As part of onboarding, you should have provided your **public SSH key** to DSVA. If you haven't, do that now.
-2. DSVA will add your public SSH key to the list of authorized keys.
-3. When that process is completed (24 hours), you will receive a Github notification that your key has been added. Once you receive the notification, continue with the steps below.
-    * If your key doesn't seem to be working, ask for help in the *#devsupport-external* Slack channel
+### For internal developers
 
-### Accessing SOCKS proxy from VA network
+6. Change to your working directory, clone and change to the devops repository
+```bash
+cd ~/YOUR-WORKING-DIRECTORY
+git clone git@github.com:department-of-veterans-affairs/devops.git
+cd devops
+```
 
-The `~/.ssh/config` file on your local system contains configuration to access the SOCKS proxy from the VA network (https://github.com/department-of-veterans-affairs/devops/blob/master/ssh/config#L34).
+7. Checkout a branch using your lowercase first and last name
+```bash
+git checkout -b add-key-first-last
+```
+8. Copy your public key into the `authorized_keys` folder renaming the file with your email address
+```bash
+cp ~/.ssh/id_rsa_vagov.pub ./ansible/roles/dsva-config/files/authorized_keys/YOUR@EMAIL.COM
+```
+9. Edit the `./ansible/global/config.yml` in the `devops` repo by adding your email address to the list of emails under the appropriate group (`developers:`, `devops:`, etc.)
 
-1. Run `ssh socks-va -D 2001 -N &`
-   _Note: the first time you connect to the jumpbox, ssh will prompt to ask
-   if you are sure you want to connect to a new host. You will be unable to
-   respond "yes" if ssh is in the background, so either bring it to the
-   foreground with `fg` or omit the `&` character from the above command._
+*Please try to keep this list alphebetized.*
 
-1. Follow the instructions below to configure your browser.
+10. Commit and push your changes to the `devops` repo
+```bash
+git add .
+git commit -m 'add ssh keys for first-name last-name'
+git push origin add-key-first-last
+```
 
-### Accessing SOCKS proxy from the internet
+11. Create a PR for this branch on Github and comment a link to the PR in the #vetsgov-devops channel on the DSVA Slack indicating that you need your ssh keys added to the `devops` repo.
 
-The `~/.ssh/config` file on your local system contains configuration to access the SOCKS proxy from outside the VA network (https://github.com/department-of-veterans-affairs/devops/blob/master/ssh/config#L28).
+12. Copy the SSH config from the `devops` repo to `~/.ssh`
+```bash
+cp ./ssh/config ~/.ssh/config
```

Comment:
since we're assuming they have the repo cloned, we can simplify some of these instructions with a `cp` command.